# Kata 10: Manejo de errores

## Capturas de ejecución de ejercicios en consola

Ejecución de archivo open.py para abrir archivo inexistente

![run-open-py](./Capturas/run-open_py.png)

Uso de try except desde consola, para mostrar mensaje de archivo no encontrado

![try-except](./Capturas/try-except.png)

Ejecución de archivo config.py

![run-config-py](./Capturas/run-config_py.png)

Controlando Exception desde config.py

![couldnt-find](./Capturas/couldnt-find.png)

Mostrando mensaje de que es un directorio y no se puede leer

![is-a-directory](./Capturas/is-a-directory.png)

Utilizando FIleNotFoundError y PermissionError, después de borrar config.txt

![permissionError](./Capturas/PermissionError.png)

Utilizando FileNotFoundError as err para mostrar mensaje de error al usuario

![fileNotFoundError-as-err](./Capturas/FileNotFoundError-as-err.png)

Utilizando OSError y su atributo errno para mostrar mensajes de error

![OSError](./Capturas/OSError.png)
